saleor/graphql/product/types.py

```diff
@@ -216,10 +216,26 @@ def resolve_products(self, info, **kwargs):
             user=user).filter(product_type=self).distinct()
 
 
+class ImageWithThumbnail(graphene.ObjectType):
```

Comment:
Shouldn't products use this type as well? I think `Image` would be a better name, Shopify has a similar type and it contains the `originalUrl`, a `transformedUrl` (thumbnail), `id` and `altText`.

Reply:
+1 for `Image`

Reply:
Ideally this should be the common interface for all the images exposed by the API, including product images.